src/main/java/appeng/client/texture/FlippableIcon.java

```diff
@@ -36,87 +38,87 @@ public FlippableIcon( final IIcon o )
 			throw new IllegalArgumentException( "Cannot create a wrapper icon with a null icon." );
 		}
 
-		this.original = o;
-		this.flip_u = false;
-		this.flip_v = false;
+		this.setOriginal( o );
```

Comment:
well you have to pay attention else you will use the non final methods on a not initialized class. Can be more error prone. If a setter is doing multiple things, they are no real setters anymore.